composeApp/src/commonMain/kotlin/com/github/familyvault/services/IChatService.kt

```diff
@@ -11,6 +11,7 @@ interface IChatService {
     fun retrieveLastMessage(chatThreadId: String): ChatMessage?
     fun sendMessage(chatThreadId: String, messageContent: String, respondToMessageId: String)
     suspend fun createGroupChat(name: String, members: List<FamilyMember>): ChatThread
+    suspend fun updateGroupChat(thread: ChatThread, members: List<FamilyMember>, newName: String?)
```

Comment:
Gdzieś tutaj przypisujesz w grupie chatu `ChatThreadType` na `Group`? Jeżeli nie, to chyba lepiej nazwać tę metodkę `updateChatThread`